Topics/User_Experience.md

```diff
@@ -31,7 +31,8 @@ ___ Nielsen's 10 Usability Heuristics___
 
 There are some things that have become a crucial part in making a software: Universal Design Principles. 
 ___Accessiblity Features___
-- [Universal Design Principles](https://www.buffalo.edu/access/help-and-support/topic3/universaldesignprinciples.html): This article provides the 7 Universal Design Principle that makes a software accessible to all users, which plays a important role in diversifying the User Experience. 
+- [Universal Design Principles](https://www.buffalo.edu/access/help-and-support/topic3/universaldesignprinciples.html): Universal Design Principles are not only used for software, but can be incorporated generally as well. These 7 principles ensure accessiblitly, consistenty, make software user-friendly. 
```

Comment:
Grammatical error: Change "These 7 principles ensure accessiblitly, consistenty, make software user-friendly." to "These 7 principles ensure accessibility, consistency, user-friendly software.

Reply:
Done